CHANGELOG.md

```diff
@@ -55,6 +49,12 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
   Most user code will be unaffected; to fix possible compiler errors you need to add an empty template parameter list to
   the respective instances (e.g. change `input_file_validator` to `input_file_validator<>`).
 
+#### CMake
+
+* [find_package](https://cmake.org/cmake/help/latest/command/find_package.html#version-selection) now accepts minimum
+versions (e.g. `find_package(SEQAN3 3.1.0)` requires at least seqan3 with a version of `>= 3.1.0` and `< 4.0.0`).
```

Comment:
We should actually not use 3.1.0 here, even if it is just an example, because we don't have a 3.1.0 release yet. Better use 3.0.1, which would at least be correct when we release it. What do you think?